File: src/test/java/shopping/wishlist/domain/WishlistFixture.java
package shopping.wishlist.domain;

public class WishlistFixture {
    public static final Long VALID_MEMBER_ID = 1L;

    public static Wishlist createWish(Long id, Long productId) {
        Wishlist wishlist = new Wishlist(VALID_MEMBER_ID, productId);
        wishlist.assignId(id);
        return wishlist;
    }
}

File: src/test/kotlin/shopping/wish/fixture/WishListFixture.kt
package shopping.wish.fixture

import shopping.member.domain.Member
import shopping.member.fixture.MemberFixture
import shopping.wish.domain.WishList

enum class WishListFixture(
    val member: MemberFixture,
) {
    `고객 1의 위시 리스트`(MemberFixture.`고객 1`)
    ;

    fun `엔티티 생성`(): WishList = WishList(member.`회원 엔티티 생성`().id)
    fun `엔티티 생성`(member: Member): WishList = WishList(member.id)
}
